async_fuse/src/fuse_request.rs

```diff
@@ -826,4 +826,10 @@ mod tests {
             _ => panic!("unexpected request operation"),
         }
     }
+
+    #[test]
```

Comment:
What's this test for?

Reply:
> What's this test for?
To fix this, it is easy to add a test case, so I don't use #[allow(dead_code)]
warning: method is never used: `len`
  --> async_fuse/src/fuse_request.rs:17:5
   |
17 |     pub fn len(&self) -> usize {
   |     ^^^^^^^^^^^^^^^

Reply:
Will test remove clippy unused warning?

Reply:
Just tested. It won't. Will submit a new one.